### ISSUE TITLE
Release to crates.io

### ISSUE THREAD

**Issue by neysofu** [Sovereign-Labs/nmt-rs#17]
We can't publish `celestia` to https://crates.io until all of its dependencies have been as well, and this includes `nmt-rs`.

**Comment by zvolin** [comment]
Hey, sorry to interrupt. I guess when you mean `celestia`, it is about your [celestia-adapter](https://github.com/Sovereign-Labs/sovereign-sdk/blob/nightly/adapters/celestia/Cargo.toml) for sovereign-sdk.
Would you reconsider renaming it before to reflect the fact that it's an adapter? At @eigerco we're developing the [celestia node](https://github.com/eigerco/celestia-node-rs) and thus we hoped we could have `celestia` name for it's CLI, like the go version does

**Comment by neysofu** [comment]
Hey @zvolin, no problem. I wasn't aware of the potential name conflict. Consider `celestia` yours, we'll switch to `sov-celestia-adapter`.

**Comment by zvolin** [comment]
Thanks @neysofu!

**Comment by zvolin** [comment]
hey, do you still plan to publish? we're considering publishing our crates and we also depend on nmt-rs

**Comment by zvolin** [comment]
ah by the way our publish to crates.io is also a blocker for you to publish `sov-celestia-adapter` :thinking:

**Comment by neysofu** [comment]
Hey, we didn't have an immediate need for it so we just postponed the release. I'll make sure we get the release out within this week, I'll ping you here when it's ready!

You folks can then release `celestia-node-rs` crates and `sov-celestia-adapter` will also be unblocked. 🙂

**Comment by neysofu** [comment]
Here it is: https://crates.io/crates/nmt-rs
